File: translations/codenet/deepseek_commented/C++2P/codellama/Code/s259332673.py
#include <iostream>

#define rep(i,n) for(int i=0;i<n;i++)

using namespace std;

int main()
{
    # Declare integer variables to store input values
    h1, h2, m1, m2, k = 0, 0, 0, 0, 0
    
    # Read input values for h1, m1, h2, m2, and k from the standard input
    h1, m1, h2, m2, k = input().split()
    
    # Calculate the difference in minutes between h2:m2 and h1:m1
    ans = (h2 * 60 + m2) - (h1 * 60 + m1)
    
    # Print the result of subtracting k from the calculated difference
    print(ans - k)
    
    return 0
}

File: translations/codenet/deepseek_commented/C++2Go/deepseek/Code/s259332673.go
package main

import (
	"fmt"
)

func main() {
	// Declare integer variables to store input values
	var h1, h2, m1, m2, k int
	
	// Read input values for h1, m1, h2, m2, and k from the standard input
	fmt.Scan(&h1, &m1, &h2, &m2, &k)
	
	// Calculate the difference in minutes between h2:m2 and h1:m1
	ans := (h2 * 60 + m2) - (h1 * 60 + m1)
	
	// Print the result of subtracting k from the calculated difference
	fmt.Println(ans - k)
}

